### ISSUE TITLE
Coordinate with JKU Institut Integriert Studieren

### ISSUE THREAD

**Issue by Yolgie** [boudicca-events/boudicca.events#294]
Try to open communication with the JKU Institut Integriert Studieren (https://www.jku.at/institut-integriert-studieren/) to enable exchange about:

- A11Y needs that should be considered
- information / tags that should be added / enriched
- events / sources that should be added
- support for user tests and feedback
- maybe future collaboration

**Comment by mahdikhashan** [comment]
I had a conversation with prof. Misenberger today, head of  the Integriert institute, they will probably help us with a11y testing!

I will have an appointment with him in coming weeks.
